openmc/plotter.py

```diff
@@ -101,9 +98,6 @@ def plot_xs(this, types, divisor_types=None, temperature=294., data_type=None,
         multi-group data.
     divisor_orders : Iterable of Integral, optional
         Same as orders, but for divisor_types
-    **kwargs
```

Comment:
Instead of removing this, why not pass it to `plt.subplots(...)`?

Reply:
now passing kwargs on to subplot. originally I noticed it was not used by the code so I thought it could be removed